appveyor.sh

```diff
@@ -80,5 +80,5 @@ export MODEL="64"
 export MODEL_FLAG="-m64"
 
 cd /c/projects/dmd/test
-../../gnumake/make -j2 all MODEL=$MODEL MODEL_FLAG=$MODEL_FLAG LIB="../../phobos;$LIB"
+../../gnumake/make -j3 all MODEL=$MODEL ARGS="-O -inline -release" MODEL_FLAG=$MODEL_FLAG LIB="../../phobos;$LIB"
```

Comment:
On second thoughts, I would suggest ARGS="-O -inline -g -debug" as it would have a slightly higher chance of catching bugs.

Reply:
Hmm you mean better error messages on failures? I think knowing which test failed is already enough info and also `-release` does more interesting things, so isn't the chance that it will catch more bugs higher with `-release`?

Reply:
The testee here is dmd built as 64-bit COFF binary so that we can ship that at some point.
From that perspective we want to maximize coverage in dmd itself, but don't care too much about the output, other than it should be correct.
I think we definitely want to exercise the C++ backend parts, that implies `-O` and `-g`.
Whether or not `-release` or `-debug` hardly matters, as it has small impact on codegen, and we already know from the Win32/64 auto-test that dmd can generate working COFF code.

In fact the output of Win32/64 cross-compilation and Win64/64 should be identical, no?
Not sure what the state of reproducible builds on Windows is, but if we have a chance of comparing builds from the 2 host compilers, that would be the most ensuring test. Even though a hard to debug one if sth. differs.

Reply:
With `-release`, we disable 99% of the tests as no asserts are checked anymore. `-debug` doesn't have an impact because the debug condition is hardly ever used.
`-g` makes sense as it covers debug info generation.